src/theia-yeoman-backend-plugin.ts

```diff
@@ -0,0 +1,38 @@
+/*********************************************************************
+* Copyright (c) 2018 Red Hat, Inc.
```

Comment:
as you're doing a refactoring, copyright date needs to be updated

Reply:
Done